### DISCUSSION TITLE
Markdown Oxide v0.22.0 status discussion

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#134] [General]
Discussion related to the status update in https://github.com/Feel-ix-343/markdown-oxide/releases/tag/v0.22.0

**Comment by Idobenhamo** [comment]
There is a way to open the daily note as an lsp command or something like this in helix?

**Reply by Feel-ix-343** [reply]
Hey!

Unfortunately not yet. Tracked here #115

**Reply by Idobenhamo** [reply]
I am runing the new version of the lsp. I just noticed 2 lsp-workspace-command are available.
![image](https://github.com/Feel-ix-343/markdown-oxide/assets/135357971/3915366d-ea22-4a38-8610-d24cd6f448f2)
The first dosent seem to do anything.
As for the second one (jump), it opens a daily note with the current day. Can you try to replicate it?
`:lsp-workspace-command`
Thank you for evreything!

**Reply by Feel-ix-343** [reply]
So happy to help!

And neat yea I just found that too. 

I am currently adding Today, Tomorrow, and Yesterday as lsp-workspace-commands => you will be able to select them in helix. You will notice that the jump command simply takes you to Today's note; these will support more days. 

Unfortunately, I don't think we will be able top support natural language because I believe helix doesn't support command arguments; I opened an issue for this here https://github.com/helix-editor/helix/issues/11079 . I may just implement it myself if possible.

**Reply by Idobenhamo** [reply]
Thank you! That would be amazing, cant wait for this feature to arrive to the lsp-workspace-commands!
I would like to suggest that maybe the Today, Tomorrow, and Yesterday files would open with first level heading of the date using the date format. For example `Today -> #2024-07-03` at the start of the file.

**Reply by Feel-ix-343** [reply]
Hey its complete. I will publish a new version. Let me know what you think. There are commands `last monday`, `last tuesday`, ... and `next monday`, ... and `today`, `tomorrow`, `yesterday`

**Reply by Feel-ix-343** [reply]
And here is an issue for your suggestion: #137  Thank you!

**Reply by Idobenhamo** [reply]
King!
Just tried to update, it is failed to compile from source.

**Reply by Feel-ix-343** [reply]
try rust nightly. Goodluck
